go/arrow/README.md

```diff
@@ -0,0 +1,155 @@
+Arrow
```

Comment:
Change this to "Apache Arrow" (or Apache Arrow for Go). This file needs a copyright header, see https://raw.githubusercontent.com/apache/arrow/master/README.md for example